.github/ISSUE_TEMPLATE/new-ruleset.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New ruleset
+about: Request/submit a new ruleset 
+---
+
+[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
+[//]: # (We welcome input from users on improving this project.)
+[//]: #
+[//]: # (Please help us by following this issue template.)
+[//]: # (You can delete all blank lines and all lines starting)
+[//]: # (with the comment marker, such as this one.)
+[//]: #
+[//]: # (Delete all but one of the following "Type" lines.)
+[//]: # (Leave in the line that best describes the issue you are reporting.)
+[//]: #
+[//]: # (If you are submitting a new ruleset, please check the list at)
+[//]: # (https://www.eff.org/https-everywhere/atlas/index.html and the open)
+[//]: # (issues and pull requests to make sure it doesn't already exist.)
+
+Type: new ruleset
+
+[//]: # (Include any other relevant information below. Thank you again for)
+[//]: # (helping to improve HTTPS Everywhere.)
```

Comment:
I guess `Domain:` belongs here more.

Reply:
I added on `new-ruleset.md`. Should I remove from `ruleset-issue.md`?

Reply:
Common sense says me "yes", but I'm not sure. I guess we should just comment it out and say in comments that it should be used when yet another domain should be added to the existing ruleset.